weed/pb/master.proto

```diff
@@ -79,6 +79,7 @@ message HeartbeatResponse {
   string metrics_address = 3;
   uint32 metrics_interval_seconds = 4;
   repeated StorageBackend storage_backends = 5;
+  bool has_duplicated_directory = 6;
```

Comment:
can it show which directory is duplicating?

Reply:
If needed, I can add a new parameter in struct HeartbeatResponse. Master can send the duplicated uuid to  volume server, then we can figure out the duplicating directory . However currently we only get one duplicating directory once at a time.

```
message HeartbeatResponse {
  uint64 volume_size_limit = 1;
  string leader = 2;
  string metrics_address = 3;
  uint32 metrics_interval_seconds = 4;
  repeated StorageBackend storage_backends = 5;
  bool has_duplicate_directory = 6;
 string duplicate_directory = 7;
}
```

Reply:
Done
current struct:
```
message HeartbeatResponse {
  uint64 volume_size_limit = 1;
  string leader = 2;
  string metrics_address = 3;
  uint32 metrics_interval_seconds = 4;
  repeated StorageBackend storage_backends = 5;
  repeated string duplicated_uuids = 6;
}

```